### PR TITLE
chore(helm): Allow configuring a custom nodePort in helm chart values

### LINKED ISSUE #1249: Allow configuring custom nodePort in helm chart
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Feature description

The current cerbos Helm chart does not allow configuring a custom value for the nodePort if the service is of `NodePort` type.
It would be helpful to be able to configure this if we want to associate a static NodePort to the service.

### What would the ideal solution look like to you?

Add a nodePort entry in the default values.yaml file for both http and grpc ports

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by stevenbressey**
#### Description

Add nodePort values in default values.yaml file to allow users to configure a custom NodePort

Fixes #1249 

#### Checklist 

- [x] The PR title has the correct prefix 
- [x] PR is linked to the corresponding issue
- [x] All commits are signed-off (`git commit -s ...`) to provide the [DCO](https://developercertificate.org/)

### CODE REVIEW

File: deploy/charts/cerbos/values.yaml [hunk 1]
```diff
@@ -85,8 +85,12 @@ envFrom: []
...
 service:
   type: ClusterIP
-  httpPort: 3592
-  grpcPort: 3593
+  http:
+    port: 3592
+    nodePort: 103592
+  grpc:
+    port: 3593
+    nodePort: 103593
```
[the remark this answers is not in the document]
Comment (stevenbressey) [reply deploy/charts/cerbos/values.yaml]: ok I rollbacked to the previous format

**Comment by charithe** [comment]
Thank you for the PR. Please can you signoff your commits to provide the [DCO](https://developercertificate.org) and push again? 

```
git rebase HEAD~2 --signoff
git push --force-with-lease origin feat/helm-configure-nodeport
```

**Comment by stevenbressey** [comment]
@charithe Thanks for the tip. I added the DCO to my first commit

**Review by charithe — APPROVED** [review]
LGTM. Thank you!

**Pull request merged**